db/schema.rb

```diff
@@ -10,12 +10,21 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_11_17_033947) do
+ActiveRecord::Schema.define(version: 2021_12_02_054844) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
   enable_extension "plpgsql"
 
+  create_table "keywords", force: :cascade do |t|
+    t.string "keyword"
```

Comment:
We added null constraints, but it has not been reflected here, I think, you need to run the migration again

Reply:
Thanks! I didn't notice that!
Fixed in [`7631238` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/76312381da15a3af06357cf7e84fd9045d11472c)